test/e2e/data/onnx_input.jpg

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html><html lang="en"><head><link href="https://nodeassets.nbcnews.com/assets/fonts/shared/fontface.css" rel="stylesheet"/><link href="https://nodeassets.nbcnews.com/_next/static/css/aa41b7c4758efd2d1d3d.css" rel="stylesheet"/><style>@font-face{font-family:iconfont;src:url(https://nodeassets.nbcnews.com/_next/static/assets/fonts/iconfont.267d40d3c6.woff)format("woff"),url(https://nodeassets.nbcnews.com/_next/static/assets/fonts/iconfont.e9d457484e.woff2)format("woff2");}</style><link href="https://nodeassets.nbcnews.com/assets/header-footer/1.8.2/news/header.css" rel="stylesheet"/><script>var _HFS = { version: "1.8.2", host: "https://nodeassets.nbcnews.com/assets/header-footer", social: "nbcnews" }</script><script>/* eslint-disable */
```

Comment:
this looks like html not jpg

Reply:
right, I removed it in https://github.com/kubeflow/kfserving/pull/1097/commits/5ef9f28ead8c7d02bd0646759bdded68bf45800d in favor of just using the iris model for the e2e test